README.md

```diff
@@ -25,7 +20,27 @@ ember install ember-frost-login
 Coming soon
 
 ## Examples
-Coming soon
+```handlebars
+{{#frost-login logo='myLogo' branding-strip='myBrand' on-enter='submitLogin' as |section|}}
```

Comment:
is it because it'll be passed on as a string to the icon attr in the addon and the dash will mess things up?